### ISSUE TITLE
Apache documentation misses access control to static files

### ISSUE THREAD

**Issue by qosobrin** [sympa-community/sympa-community.github.io#11]
I have just finished a fresh sympa installation and, when I first tried the web interface, it failed serving  css, icons and any other static files due to "access denied" errors. I used the configuration settings explained in https://github.com/sympa-community/sympa-community.github.io/blob/master/manual/install/configure-http-server-apache.md

I implemented access controls to the static-sympa directory and they worked fine. I think these settings should be added to that installation manual page:

```
<Directory $STATICDIR>
    Require all granted
</Directory>
```
On a similar wave the same could be said for the Apache 2.2 configuration file (though I haven't tried it):

```
<Directory $STATICDIR>
    Order deny,allow
    Allow from all
</Directory>
```

On a different matter, but also related to this page, I do not see the point in showing settings that need to be changed immediately. I think this block in the Apache 2.4 configuration:

```
<Location /sympa>
    SetHandler fcgid-script

    # Don't forget to edit lines below!
    Require all denied
    #Require all granted
</Location>
```

should be changed to something like this:

```
<Location /sympa>
    SetHandler fcgid-script

    # Interchange the commented line below to restrict access
    #Require all denied
    Require all granted
</Location>
```

and the same could be said about the Apache 2.2 configuration.

**Comment by ikedas** [comment]
Hi @qosobrin,

`denied` access is intended. If the access was granted by default, as soon as you started services, it could be accessible from all over the world (Of course your machine must be proteced by firewall... However, humans are creatures that are born to make mistakes).

However, I agree to that `denied` restriction is overkill. [Recent configuration bundled in RPM](https://github.com/ikedas/sympa-rpm/blob/master/sympa-httpd24-fcgid.conf) adopts [`Require local`](https://httpd.apache.org/docs/trunk/mod/mod_authz_host.html#reqlocal) as default setting.  By this setting, you may test new installation using a browser in local host.

**Comment by ikedas** [comment]
Fixed in d418a99. Close.
